.github/workflows/test.yml

```diff
@@ -175,12 +173,3 @@ jobs:
         with:
           name: Cypress tests
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_TEST_REPORTER_WEBHOOK }}
-
-  # Included as a single job to check for cypress-test-matrix success, as a matrix cannot be checked.
-  cypress-tests:
```

Comment:
qq: Could we still enforce all cypress tests passing before a PR is merged without this check?

Reply:
yes, we are now enforcing this through `cypress:e2e`, instead of through `cypress-tests`.